_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -540,14 +564,14 @@ def perform_requests(self):
                         if request["harvest"]["link_text"] in link.get_text():
                             request_url = link.get("href")
                             break
-                        logger.error(f"Unable to harvest url {request['harvest']}")
-                        self.interrupt()
 
                 if harvest_details["type"] == "name":
                     for name in harvest_details["names"]:
                         input_name = soup.find("input", attrs={"name": name})
                         input_value = input_name.attrs.get("value")
                         harvest_dict[name] = input_value
+                if harvest_details["type"] == "form":
+                    request_url = soup.find("form", id="surveys_upload_form").get("action")
```

Comment:
```suggestion
                    request_url = soup.find("form", id=harvest_details["id"]).get("action")
```
To make it more generic and use the config?